src/frontend/Komponenter/Oppgavebenk/OppgaveRad.tsx

```diff
@@ -85,14 +80,14 @@ const OppgaveRad: React.FC<Props> = ({ oppgave, mapper, settFeilmelding }) => {
         feilmelding,
     } = useOppgave(oppgave);
 
-    const [anker, settAnker] = useState<HTMLElement>();
+    const [anker, settAnker] = useState<Element | null>(null);
```

Comment:
Gjøres dette for å sette den eksplisitt til null, i stedet for undefined?

Reply:
Sånn det var før så var vel feil å si at typen er `HTMLElement` men så instansiere `anker` til `undefined`, så det blir mer riktig nå :) 

Svaret på spørsmålet ditt er altså ja :p